### DISCUSSION TITLE
Local UI

### DISCUSSION THREAD

**Discussion by abacaj** [abacaj/replit-3B-inference#12] [Ideas]
As posted in this issue: https://github.com/abacaj/replit-3B-inference/issues/10

Can discuss what use cases are needed with UI.

**Comment by 77bgh** [comment]
I was thinking:

1. An input area where the user to input his question
2. A result area where the user to receive the answer 
3. A history area where the user to see the previous questions and corresponding answers 
4. A file input function where from the user can upload different file types to be indexed for potential question requests 
5. A separate input area where the user to put questions and receive answers considering only the documents he uploaded

**Comment by 77bgh** [comment]
something like this 

![image](https://github.com/abacaj/replit-3B-inference/assets/64399919/2cd29de8-f4c3-4235-9cfe-03cb65f5c408)

still unclear why in the answer it gives as well details like:

<|endoftext|># Changelog

"## [1.1.0] - 2020-11-23
"### Added
"- Added ability to add a description to the package.
"- Added ability to add a homepage to the package.
"- Added ability to add a keywords to the package.


this is the code I used , just a GUY,py

[https://github.com/77bgh/GUI-for-chatGBT/blob/main/GUI.py](url)

[GUI.py.txt](https://github.com/abacaj/replit-3B-inference/files/11930111/GUI.py.txt)


looks slower than in anaconda prompt, but higly acceptable.
